src/main/java/org/jabref/model/entry/AuthorList.java

```diff
@@ -169,6 +171,7 @@ public static AuthorList parse(final String authors) {
         if (authorList == null) {
             AuthorListParser parser = new AuthorListParser();
             authorList = parser.parse(authors);
+            authorList.authorsUnformatted = new String(authors);
```

Comment:
I don't think this is avoidable with the current structure if we want to cache unformatted latex-free strings.

Reply:
(see above)